back/package.json

```diff
@@ -8,9 +8,10 @@
     "start:dev": "nodemon --exec babel-node --extensions \".ts\" src/index.ts",
     "start:debug": "run-p -l type-check:watch \"start:dev -- --inspect-brk\"",
     "start:prod": "node ./dist/index.js",
-    "start:mock-repository": "cross-env MOCK_REPOSITORY=true npm run start",
-    "start:local-db": "docker-compose up || echo \"Run docker-compose up manually\"",
-    "stop:local-db": "docker-compose down || echo \"Run docker-compose down manually\"",
+    "start:mock": "cross-env MOCK_REPOSITORY=true npm run start",
+    "start:localdb": "cross-env run-p -l start start:docker-db",
```

Comment:
Check this [package.json](https://github.com/Lemoncode/planning-poker-example/blob/master/back/package.json) on planning poker. We usually use always `npm start` as main command to run with local-db because it will be the main execution. And `npm run start:mock` as secondary command.

Reply:
Done and pushed